src/ContentHub.php

```diff
@@ -214,6 +214,23 @@ public function deleteEntity($uuid)
     }
 
     /**
+     * Purges all entities from the Content Hub.
+     *
+     * This method should be used carefully as it deletes all the entities for
+     * the current subscription from the Content Hub.
+     */
+    public function purge() {
+        $list = $this->listEntities();
```

Comment:
just a question. looks like we have 4 spaces here for indentation. Are we using a different coding standard for this PHP client library?